ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -370,6 +370,8 @@ def serialize(self) -> JsonObject:
             except Exception as e:
                 self.add_error(e)
 
+        type_definitions = getattr(self.display_context, "type_definitions", [])
```

Comment:
we shouldn't need this attribute in display context. We should be able to iterate through the node output displays in order to generate the `"type_definitions"` field